appium.js

```diff
@@ -73,12 +91,91 @@ function connectGetDefaultArgs () {
   });
 }
 
+function connectStartSession (win) {
+  ipcMain.on('start-session', () => {
+    let sessionWin = new BrowserWindow({width: 800, height: 600, webPreferences: {devTools: true}});
+    let sessionHTMLPath = path.resolve(__dirname, 'app', 'index.html#/session');
+    sessionWin.loadURL(`file://${sessionHTMLPath}`);
+    sessionWin.show();
+
+    // When you close the session window, kill the associated Appium session
+    let sessionWinID = sessionWin.webContents.id;
+    sessionWin.on('closed', () => {
+      killSession(sessionWinID);
+      sessionWin = null;
+    });
+
+    // When the main window is closed, terminate the appium session and close the session window
+    win.once('closed', () => {
+      // sessionWin.close();
```

Comment:
I have another branch that I'm working on that fixes this problem. I'll just remove it from this PR for now and it'll be in a future PR.